brianlong/www.validators.app
# 180011190 fix max value on speedometers

#### What's this PR do?
- Fixes UI error showed on the screenshot in PT

#### How should this be manually tested?
- The quickest way to reproduce (either locally, or on staging) is to override automatically calculated ASN score:
```AsnStat.last.update_columns(average_score: 11)```
Then check the result in the browser, in data centers view, sorted by ASN.

#### What are the relevant tickets?
- https://www.pivotaltracker.com/n/projects/2177859/stories/180011190

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
